Stratis.Bitcoin/Features/BlockStore/LoopSteps/BlockStoreStepFindBlocksTask.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
+{
+    /// <summary>
+    /// Find blocks to download by asking the BlockPuller
+    /// 
+    /// Once the BatchDownloadSize has been reached
+    /// the task will be removed from the routine via context.StopFindingBlocks()
+    /// </summary>
+    public sealed class BlockStoreStepFindBlocksTask : BlockStoreStepTask
```

Comment:
You can probably remove the word Task, maybe  better name is InnerStep so StoreDownloadInnerStepFindBlocks